Gems/Atom/Component/DebugCamera/Code/Include/Atom/Component/DebugCamera/CameraComponent.h

```diff
@@ -102,25 +104,41 @@ namespace AZ
             void SetFrustumHeight(float height) override;
             void SetOrthographic(bool orthographic) override;
             void SetOrthographicHalfWidth(float halfWidth) override;
+            void SetStereoscopicView(const AZ::Quaternion& viewQuat, uint32_t xrViewIndex) override;
```

Comment:
Maybe rename this to a different name? 
It's a bit confusion since "GetStereoscopicView" is not related to this "SetStereoscopicView"